src/MeetingsSDKAdapter.js

```diff
@@ -0,0 +1,153 @@
+import {MeetingsAdapter} from '@webex/component-adapter-interfaces';
+import {Observable, fromEvent, concat} from 'rxjs';
+import {filter, finalize, map, publishReplay, refCount} from 'rxjs/operators';
+
+const LOCAL = 'local';
+const MEDIA_READY = 'media:ready';
+
+/**
+ * The `MeetingsSDKAdapter` is an implementation of the `MeetingsAdapter` interface.
+ * This adapter utilizes the Webex JS SDK to create and join webex meetings.
+ *
+ * @class MeetingsSDKAdapter
+ * @extends {MeetingsAdapter}
+ */
+export default class MeetingsSDKAdapter extends MeetingsAdapter {
+  constructor(datasource) {
+    super(datasource);
+    this.meetingsObservables = {};
```

Comment:
For consistency with the rooms and people adapters, let's go with:
```suggestion
    this. getMeetingObservables = {};
```